src/CLU/Microsoft.Azure.Commands.Resources/RoleAssignments/NewAzureRoleAssignmentCommand.cs

```diff
@@ -101,6 +104,7 @@ public class NewAzureRoleAssignmentCommand : ResourcesBaseCmdlet
         [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, ParameterSetName = ParameterSet.ResourceWithSPN,
             HelpMessage = "Parent resource of the resource to assign the role to, if there is any.")]
         [ValidateNotNullOrEmpty]
+        [Alias("parent", "p")]
```

Comment:
@hovsepm I think this is too easy to confuse with the other -p (property) - we shoudol try to be consistent with the single character parameters, they should be applicable at least across an RP.